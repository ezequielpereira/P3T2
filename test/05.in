test_tiempo
10000000-10000-1-20-20
Fin
