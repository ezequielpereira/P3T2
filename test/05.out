1>Cantidad: 10000
2>Fin.
